node/network/approval-distribution/src/lib.rs

```diff
@@ -113,6 +113,8 @@ impl Knowledge {
 	}
 }
 
+type MissingKnowledge = HashSet<MessageFingerprint>;
```

Comment:
nit: A small descriptive comment would be appreciated, for this and the following type `PeerKnowledge`